# How does volatility impact returns - part 2
In part 1 of "How does volatility impact returns?", we looked at scenarios where the mean return was the same, but volatility (standard deviation) was different. The conclusions were that:

- Volatility does not have an impact on expected returns
- But volatility does impact the probability of reaching various financial goals

However, in the real world, lower volatility often results in lower average returns:

"When volatility increases, risk increases and returns decrease. Risk is represented by the dispersion of returns around the mean." (http://www.investopedia.com/articles/financial-theory/08/volatility.asp#ixzz4KG4pag92)

So in this part 2, we'll look at whether its better to invest in higher-risk/higher average return funds, or lower-risk/lower average return funds.

### So what's the relationship between mean returns and volatility?

According to empirical analysis of Vanguard's mutual funds (see vanguard_fund_analysis), here is an equation to relate mean return with expected mean standard deviation:

In [1]:

def investment_mean_to_std(mean_return):
    """ Given the mean return, output the standard deviation return. """
    slope = 3.4856115546738726
    intercept = 2.2961232860559
    return slope * mean_return + intercept

In [2]:
mutual_fund_mean_returns = [10, 8, 6, 4]
mutual_fund_std_returns = [investment_mean_to_std(mean_return) for mean_return in mutual_fund_mean_returns]
mutual_fund_std_returns

[37.15223883279463, 30.18101572344688, 23.209792614099133, 16.23856950475139]

Okay, so now we have some good numbers to go on to build some hypothetical funds to analyze...

### Hypothetical Funds
- Fund 1: return mean: 10%, standard deviation: 37.15%
- Fund 2: return mean: 8%, standard deviation: 30.18%
- Fund 3: return mean: 6%, standard deviation: 23.21%
- Fund 4: return mean: 4%, standard deviation: 16.24%

And again, we will be contributing $5,500 per year to these funds, and running the simulation for 30 years.